CsvParser/csvParser_test.go

```diff
@@ -35,7 +35,7 @@ func Test_CsvParserEntries(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			incidents := CsvParser(tt.input)
+			incidents := CsvParser(tt.input, 20)
```

Comment:
Why is 20 chosen here? Is it arbitrary? If so, please add a comment saying so since it does not match the number of incidents in incidentfile.csv

Reply:
I removed 20!